### ISSUE TITLE
Training/inference speed is slow.

### ISSUE THREAD

**Issue by vandesa003** [zylo117/Yet-Another-EfficientDet-Pytorch#210]
Hi @zylo117 , thanks for your great work! I try to use your implementation for my customised dataset. While the model is training I found the GPUs cannot be fully used, only ~10% usage and I can only get ~2 steps per second. I guess that mostly because low efficiency of pytorch dataloader, but do you have any idea to speed up the training?
<img width="558" alt="Screenshot 2020-04-27 at 1 35 40 AM" src="https://user-images.githubusercontent.com/10252812/80314991-68b4c180-8827-11ea-8b74-de61fe7c9446.png">

**Comment by zylo117** [comment]
you can increase batchsize or implement a prefetch dataloader

**Comment by vandesa003** [comment]
Thanks for your advice, I think a prefetch dataloader should solve this problem. Do you have any idea how to implement it? any hints is good to hear.

**Comment by vandesa003** [comment]
Another reason I think is because too many CPU operations in loss calculation.
